"\n" in table cell is replaced by an actual newline in the output

When you format a table that contains `\n`,  `fmt` replaces it with an actual newline instead of outputting the characters `\n` again.
Failing testcase: https://github.com/sherter/ghokin/commit/d6f88c684fc8763261c92baae5751b4a943cd733

```
--- FAIL: TestTransform (0.00s)
    --- FAIL: TestTransform/fixtures/escape-newline.feature (0.00s)
        transformer_test.go:471:
                Error Trace:    /home/simon/repositories/ghokin/ghokin/transformer_test.go:471
                Error:          Not equal:
                                expected: "Feature: Test\n\n  Scenario: Newline in table value\n    Given I have a table:\n      | a | b \\n c | \\n d | \\n e \\n |\n      | f | g      | h    | i       |\n"
                                actual  : "Feature: Test\n\n  Scenario: Newline in table value\n    Given I have a table:\n      | a | b \n c | \n d | \n e \n |\n      | f | g     | h   | i     |\n"

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -4,4 +4,8 @@
                                     Given I have a table:
                                -      | a | b \n c | \n d | \n e \n |
                                -      | f | g      | h    | i       |
                                +      | a | b
                                + c |
                                + d |
                                + e
                                + |
                                +      | f | g     | h   | i     |

                Test:           TestTransform/fixtures/escape-newline.feature
FAIL
exit status 1
FAIL    github.com/antham/ghokin/v3/ghokin      0.113s
```

According to https://cucumber.io/docs/gherkin/reference/#table-cell-escaping it's valid to use `\n` in table cells.

--- a/ghokin/fixtures/escape-new-line.feature
+++ b/ghokin/fixtures/escape-new-line.feature
@@ -1,0 +1,6 @@
+Feature: Test
+
+  Scenario: Newline in table value
+    Given I have a table:
+      | a | b \n c | \n d | \n e \n |
+      | f | g      | h    | i       |

--- a/ghokin/transformer.go
+++ b/ghokin/transformer.go
@@ -217,7 +217,10 @@ func extractTableRows(tokens []*gherkin.Token) []string {
 		for _, data := range tab.Items {
 			// A remaining pipe means it was escaped before to not be messed with pipe column delimiter
 			// so here we introduce the escaping sequence back
-			row = append(row, strings.ReplaceAll(data.Text, "|", "\\|"))
+			text := data.Text
+			text = strings.ReplaceAll(text, "|", "\\|")
+			text = strings.ReplaceAll(text, "\n", "\\n")
+			row = append(row, text)
 		}
 
 		rows = append(rows, row)

--- a/ghokin/transformer_test.go
+++ b/ghokin/transformer_test.go
@@ -437,6 +437,10 @@ func TestTransform(t *testing.T) {
 			"fixtures/escape-pipe.feature",
 		},
 		{
+			"fixtures/escape-new-line.feature",
+			"fixtures/escape-new-line.feature",
+		},
+		{
 			"fixtures/several-scenario-following.feature",
 			"fixtures/several-scenario-following.feature",
 		},

